plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java

```diff
@@ -302,11 +306,11 @@ static boolean validateLocation(String location, WizardPage page) {
         FilePermissions.verifyDirectoryCreatable(path);
         return true;
       } catch (FileAlreadyExistsException ex) {
-          String message = MessageFormat.format(Messages.getString("FILE_LOCATION"), location); //$NON-NLS-1$
+          String message = Messages.getString("FILE_LOCATION", location); //$NON-NLS-1$
           page.setMessage(message, ERROR);
           return false;  
       } catch (IOException ex) {
-        String message = MessageFormat.format(Messages.getString("INVALID_PATH"), location); //$NON-NLS-1$
+        String message = Messages.getString("INVALID_PATH", location); //$NON-NLS-1$
```

Comment:
There are other usages of `MessageFormat` in this class. If you want, you can remove them (and the import).